externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/SchemaHelper.scala

```diff
@@ -41,6 +41,24 @@ object SchemaHelper {
    */
   final val YEAR_MONTH_INTERVAL = "YearMonthIntervalType"
 
+  /**
+   * SPARK-7768: UDT API has been public since Spark 3.2.0.
+   * Here we distinguish the UDT if the datatype is subClass of UserDefinedType
+   * that is available in public since Spark 3.2.0
+   */
+  final private val isUserDefinedType = (dt: DataType) => {
+    val dataType = "org.apache.spark.sql.types.DataType"
+    // Canonical name of Spark 3.2.0 DataType UserDefinedType.
+    val userDefinedType = "org.apache.spark.sql.types.UserDefinedType"
+
+    var curClass: Class[_] = dt.getClass
```

Comment:
```suggestion
    var curClass: Class[_] = dt.getClass
```
`currentClass` is more prefered as `cur` is not common abbreviation for `current`

Reply:
Sure, thank you.